--- pifm.c	2014-01-08 23:11:35.000000000 +0000
+++ pifm.new	2016-09-24 13:42:17.024508910 +0100
@@ -191,7 +191,7 @@
     };
     void consume(float* data, int num) {
         for (int i=0; i<num; i++) {
-            float value = data[i]*8;  // modulation index (AKA volume!)
+            float value = data[i]*1;  // modulation index (AKA volume!)
            
             // dump raw baseband data to stdout for audacity analysis.
             //write(1, &value, 4);
